src/components/Sidebar/SidebarTile.component.tsx

```
@@ -0,0 +1,135 @@
+                        />
+                    ))}
+            </ListItemButton>
+            {children && (
+                <Collapse in={!isCollapsed} timeout="auto" unmountOnExit>
+                    <List component="div" disablePadding>
+                        {children.map((childItem, index) => (
+                            <ListItemButton
+                                onClick={() =>
+                                    handleButtonClick(childItem.route)
+                                }
+                                role="link"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<a>
<link>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)